warehouse/query-core/src/main/java/datawave/query/iterator/logic/AndIterator.java

```diff
@@ -256,17 +258,30 @@ public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean i
             try {
                 for (NestedIterator<T> itr : child.leaves()) {
                     if (itr instanceof SeekableIterator) {
-                        ((SeekableIterator) itr).seek(range, columnFamilies, inclusive);
+                        try {
+                            ((SeekableIterator) itr).seek(range, columnFamilies, inclusive);
+                        } catch (Exception e2) {
```

Comment:
The exception here can potentially be an `IterationInterruptedException`. Is it possible to include specific handling for the exception and/or if appropriate lowering the log-levels?

Reply:
What do you have in mind?

Reply:
I need to update handling of the IteratorionInterruptedException which we explicitly catch in the QueryIterator.  We may want to more gracefully handle that exception.